sabnzbd/bpsmeter.py

```diff
@@ -243,6 +244,25 @@ def update(self, server=None, amount=0):
             self.bps = (self.bps * (self.last_update - self.start_time) + amount) / (t - self.start_time)
         except:
             self.bps = 0.0
+            self.server_bps = {}
```

Comment:
Is the reset needed?

Reply:
My assumption was that if the total bps should be reset to 0, then so should the bps for each server. Otherwise the sum of the servers and the total won't be in sync. It was easier to set it to {} than setting each server to 0.